lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1358,6 +1358,37 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value cdf = buildUnitNormalCdf(b, loc, payloadArgs[0]);
     return b.create<arith::MulFOp>(loc, payloadArgs[0], cdf);
   }
+  if (auto geluBackward = dyn_cast<AtenGeluBackwardOp>(op)) {
+    if (!geluBackward.getType()
+             .cast<ValueTensorType>()
+             .getDtype()
+             .isa<mlir::FloatType>()) {
+      geluBackward.emitError("unimplemented: non-floating point dtype");
+      return nullptr;
+    }
+    Type elementType = payloadArgs[1].getType();
+    Value constant0 = b.create<arith::ConstantOp>(
```

Comment:
nit: shall we give this a different name? When using constant0 or cst0 we usually mean 0. Same for constant1 below.